common/errors/errors_test.go

```diff
@@ -0,0 +1,83 @@
+package errors_test
+
+import (
```

Comment:
Please use `github.com/stretchr/testify`

Reply:
sure

Reply:
I see you have decided to do it your own way, by using `gotest.tools/assert`. you look like a rebel :) 

Ok Let's compare how a result looks with different tools:
"github.com/stretchr/testify/assert" (the package we decided you use for all unit tests)
```
--- FAIL: TestNew (0.00s)
    errors_test.go:15: 
        	Error Trace:	errors_test.go:15
        	Error:      	Not equal: 
        	            	expected: "foo"
        	            	actual  : "foofff"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-foo
        	            	+foofff
        	Test:       	TestNew
    errors_test.go:17: assertion failed: foo (string) != foofff (string)
FAIL
FAIL	github.com/pastelnetwork/gonode/common/errors	0.011s
FAIL
```

"gotest.tools/assert" (your choice)
```
--- FAIL: TestNew (0.00s)
    errors_test.go:16: assertion failed: foo (string) != foofff (string)
FAIL
FAIL	github.com/pastelnetwork/gonode/common/errors	0.010s
FAIL
```

Now please imagine, if we compare the large structure, how useful the detailed result can be for us.

Reply:
it's automatically imported using goimports, i didn't use choose it :)